### PR TITLE
PRT-355 Rename e2e test file

### PR DESCRIPTION
**Pull request by kajeagentspi**
* **Please check if the PR fulfills these requirements**
- [x] Commit message follows the Contribution Guidelines
- [x] Tests ran locally and added/modified if needed
- [x] Docs have been added/updated, if applicable
- [x] If applicable - JIRA ticket ID was added

* **What kind of change does this PR introduce?** (Bug fix, feature, unit tests, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Please describe what manual tests you ran, if applicable**

* **Other information**:

### CODE REVIEW

**Comment by ranlavanet** [comment]
@kajeagentspi please add the eth fixes to this pr so we can merge this to main

File: testutil/e2e/e2e.go [hunk 1]
```diff
@@ -770,7 +770,17 @@ func runE2E() {
 	lt.startJSONRPCProxy(jsonCTX)
 	lt.startJSONRPCProvider("http://127.0.0.1:1111", jsonCTX)
 	lt.startJSONRPCGateway(jsonCTX)
-	lt.checkJSONRPCGateway("http://127.0.0.1:3333/1", time.Second*30)
+	lt.checkJSONRPCGateway("http://127.0.0.1:3333/1", time.Minute*2)
```
Comment (Aleksao998) [inline testutil/e2e/e2e.go:773]: i see for other gateways we wait 30sec, why 2 minutes here?

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -71,15 +71,199 @@ jobs:
...
 
-    ######################################################
-    ### Run relayer unitests
-    ######################################################
-    - name: Run Relayer unit Tests
-      run: go test ./relayer/lavasession/ ./protocol/chaintracker/ ./relayer/chainproxy/ -v
\ No newline at end of file
+    - name: cat testutil/e2e/logs/01_stakeLava.log
+      if: steps.check_01_stakeLava.outputs.files_exists == 'true'
+      run: cat testutil/e2e/logs/01_stakeLava.log
```
Comment (Aleksao998) [inline .github/workflows/e2e.yml:95]: can you please explain what are these tests?

**Review by Aleksao998 — CHANGES_REQUESTED** [review]
The code looks good 🚀 i see that tests passed on github actions, but when i ran them locally using:
```
go test ./testutil/e2e/ -v -timeout 1200s
```
i get an panic:
```
Jan 25 13:41:14 INF Waiting JSONRPC Gateway
panic: Error found in logs

goroutine 538 [running]:
github.com/lavanet/lava/testutil/e2e.(*lavaTest).saveLogs(0xc000d8f900)
        /home/aleksaopacic/Documents/Projects/lava/testutil/e2e/e2e.go:681 +0x49e
github.com/lavanet/lava/testutil/e2e.(*lavaTest).listenCmdCommand(0xc000d8f900, 0x0?, {0x145513e, 0x31}, {0x141fb4b, 0x13})
        /home/aleksaopacic/Documents/Projects/lava/testutil/e2e/e2e.go:259 +0xa9
github.com/lavanet/lava/testutil/e2e.(*lavaTest).startJSONRPCGateway.func1()
        /home/aleksaopacic/Documents/Projects/lava/testutil/e2e/e2e.go:240 +0x3a
created by github.com/lavanet/lava/testutil/e2e.(*lavaTest).startJSONRPCGateway
        /home/aleksaopacic/Documents/Projects/lava/testutil/e2e/e2e.go:239 +0x2ed
FAIL    github.com/lavanet/lava/testutil/e2e    52.915s
FAIL
Jan 25 13:41:14 INF Waiting JSONRPC Gateway
panic: Error found in logs

goroutine 538 [running]:
github.com/lavanet/lava/testutil/e2e.(*lavaTest).saveLogs(0xc000d8f900)
        /home/aleksaopacic/Documents/Projects/lava/testutil/e2e/e2e.go:681 +0x49e
github.com/lavanet/lava/testutil/e2e.(*lavaTest).listenCmdCommand(0xc000d8f900, 0x0?, {0x145513e, 0x31}, {0x141fb4b, 0x13})
        /home/aleksaopacic/Documents/Projects/lava/testutil/e2e/e2e.go:259 +0xa9
github.com/lavanet/lava/testutil/e2e.(*lavaTest).startJSONRPCGateway.func1()
        /home/aleksaopacic/Documents/Projects/lava/testutil/e2e/e2e.go:240 +0x3a
created by github.com/lavanet/lava/testutil/e2e.(*lavaTest).startJSONRPCGateway
        /home/aleksaopacic/Documents/Projects/lava/testutil/e2e/e2e.go:239 +0x2ed
FAIL    github.com/lavanet/lava/testutil/e2e    52.915s
FAIL
```
Do you know why is this happening?

**Comment by ranlavanet** [comment]
I also get a panic

File: testutil/e2e/e2e.go (hunk 1 above)
Comment (kajeagentspi) [reply testutil/e2e/e2e.go:773]: jsonrpc which uses ETH takes 64 blocks to boot which takes a long time.

**Review by ranlavanet — APPROVED** [review]

**Pull request merged**